arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift

```diff
@@ -31,23 +31,18 @@ class DistanceMeasurementAnalysisViewController: UIViewController, AGSGeoViewTou
     
     required init?(coder: NSCoder) {
         // Create the scene.
-        scene = AGSScene(basemap: .imagery())
+        scene = AGSScene(basemap: .topographic())
         
-        /// The url of the Terrain 3D ArcGIS REST Service.
-        let worldElevationServiceURL = URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!
-        /// The url of the image service for elevation in Brest, France.
-        let brestElevationServiceURL = URL(string: "https://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer")!
+        // The elevation image service URL.
+        let elevationServiceURL = URL(string: "https://tiles.arcgis.com/tiles/d3voDfTFbHOCRwVR/arcgis/rest/services/MNT_IDF/ImageServer")!
```

Comment:
According to Tanner, we substitute the elevation source with this service. I cannot really tell if the surface is elevated 🤔  - the Brest area is pretty flat as for this elevation source. But it does fixed the flaky-looking buildings in the scene.